engine/include/gen/graphics/graphicsExceptions.hpp

```diff
@@ -12,4 +12,11 @@ namespace gen
 
 		explicit vulkan_error(const std::string & message);
 	};
+
+	struct graphics_error : public std::runtime_error
```

Comment:
Curious why these types are in snake_case?

Reply:
This was done to follow the structure of runtime_error though I've been debating if I should follow this or just follow our own style instead.

Reply:
I'm more so leaning on following our own style but have yet to take the plunge.